### PR TITLE
Feature/code quality

### PR DESCRIPTION
**Pull request by ladydascalie**
This PR contains code quality improvement for the core, and should incur no change in behaviours.
However, commit [de1df78](https://github.com/LUSHDigital/core/commit/de1df782703cfba96f301e8563f2d7d26965c6c4) does remove an element from a function signature.

No effect could be observed in tests, but I thought it best to mention it here.

### CODE REVIEW

File: middleware/internal/greeter/greeter.pb.go [hunk 1]
```diff
@@ -6,9 +6,10 @@ package greeter
 import (
 	context "context"
 	fmt "fmt"
+	math "math"
```
Comment (zeevallin) [inline middleware/internal/greeter/greeter.pb.go]: Please revert this change since this is generated code.

**Review by zeevallin — CHANGES_REQUESTED** [review]
Only a revert on the generated proto file and it'll be gucci

File: middleware/internal/greeter/greeter.pb.go (hunk 1 above)
Comment (ladydascalie) [reply middleware/internal/greeter/greeter.pb.go]: Done
